### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by silviatofana**
https://silviatofana.github.io/todolist/dist/

Project requirements
Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).
Add event listener to the checkbox (change).
Update items object's value for completed key upon user actions.
Implement a function for the "Clear all completed" button (use filter() method).
Store the updated array of items in local storage, so the user gets the correct list values after the page reloads.

### CODE REVIEW

File: src/modules/todolist.js [hunk 1]
```diff
@@ -71,5 +71,3 @@ class ToDoList {
       this.updateTasks();
```
Comment (Ismailco) [inline src/modules/todolist.js:71]: - [ ] Great work implementing the  status updates method, but you should create a new JavaScript file imported as a module, please move your code to a new file

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @silviatofana,

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by silviatofana** [comment]
Already put todolist.js and elements.js in modules folder and import them in index.js from modules

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @silviatofana, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
- [ ] I highly encourage you to commit each and every meaningful change in your project it shows that you are dedicated


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 // css files here
 import './style.css';
-import TodoList from './todolist.js';
-import * as Element from './elements.js';
+import TodoList from './modules/todolist.js';
+import * as Element from './modules/elements.js';
 
 const newTask = new TodoList();
```
[the remark this answers is not in the document]
Comment (silviatofana) [reply src/index.js:6]: - it's showing at the bottom of the list, the update list
- it works when you click on the check box
- when you click the check box it's working completed: true/false

[the remark this answers is not in the document]
Comment (silviatofana) [reply src/index.js:6]: image

[the remark this answers is not in the document]
Comment (silviatofana) [reply src/index.js:6]: kindly  run npm install

File: src/modules/todolist.js (hunk 1 above)
Comment (sajid-munawar) [reply src/modules/todolist.js:71]: @Ismailco please see project again . They are already imported as a module .

Comment (Ismailco) [reply src/modules/todolist.js:71]: You need a new file

Comment (silviatofana) [reply src/modules/todolist.js:71]: @ismailco 
just a newfile.js for you Sir with every lines of code from the same todolist?  even the project works everything well ?
https://silviatofana.github.io/todolist/dist/

File: src/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (t-bello7) [reply src/index.js:6]: @DeliceLydia can the required requirements have already been implemented. You can check the deployed link here.
https://silviatofana.github.io/todolist/dist/

**Review by vic778 — APPROVED** [review]
Hi @silviatofana   , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Charlie Brown

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification

**Comment by DeliceLydia** [comment]
Hello Next reviewer, this project should be approved I recognized that @silviatofana worked on the `add` and `remove` features which was not a requirement for this project but she explained.

**Comment by silviatofana** [comment]
thanks a lot guys that you understand my project works fine and also for let me explain to

**Pull request merged**